### PR TITLE
[ICA-1] - Implement 'Properties' endpoints

### PR DESCRIPTION
**Pull request by MiklosArpad**
### Story link: https://miklosarpad.atlassian.net/browse/ICA-1

- Implement Property model
- Generate database schema based on the model via migrations script
- Implement related Repository, Service and Controller endpoints

### CODE REVIEW

File: src/main/java/hu/ingatlancentrum/api/controller/PropertyController.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import java.util.List;
+
+@RestController
+@RequestMapping(RouteConstants.PROPERTIES)
+public class PropertyController {
+    private final PropertyService propertyService;
+
+    public PropertyController(PropertyService propertyService) {
+        this.propertyService = propertyService;
```
[the remark this answers is not in the document]
Comment (MiklosArpad) [reply src/main/java/hu/ingatlancentrum/api/controller/PropertyController.java:20]: We can do this on a separate branch when we do a general boilerplate clean up.

[the remark this answers is not in the document]
Comment (MiklosArpad) [reply src/main/java/hu/ingatlancentrum/api/controller/PropertyController.java]: Done

**Review by jbence1994 — APPROVED** [review]
LGTM

**Comment by MiklosArpad** [comment]
Follow up PR for request and response mapping: https://github.com/MiklosArpad/ingatlan-centrum-api/pull/20

**Pull request merged**